src/Parser/Warlock/Destruction/Modules/Talents/ChannelDemonfire.js

```diff
@@ -1,43 +0,0 @@
-import React from 'react';
-
-import Analyzer from 'Parser/Core/Analyzer';
-import Combatants from 'Parser/Core/Modules/Combatants';
-
-import SPELLS from 'common/SPELLS';
-import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
-import SpellIcon from 'common/SpellIcon';
-import { formatNumber, formatPercentage } from 'common/format';
-
-class ChannelDemonfire extends Analyzer {
```

Comment:
Can you explain why this module should be deleted?

Reply:
takes up too much space, could opt for a small stat box but still ultimately pretty pointless to see dps from demonfire